src/Reference/ReferencesList.tsx

```diff
@@ -0,0 +1,70 @@
+import './ReferencesList.css'
+
+import {
+    forwardRef, useEffect, useImperativeHandle,
+    useState,
+} from 'react'
+
+export default forwardRef((props: { items: Array<string>, command: ({ id }: { id: string }) => any }, ref) => {
+    const [selectedIndex, setSelectedIndex] = useState(0)
+
+    const selectItem = (index: number) => {
+        const item = props.items[index]
```

Comment:
Suggest using destructuring assignment to avoid repeating `props.` throughout this component:

```ts
const {items, command} = props;
```